ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkSanitizer.kt

```diff
@@ -142,7 +143,15 @@ class ApkSanitizer(
 
         private fun sanitizeEntry(entry: ApkEntry): AppFile {
             val name = resourceNameSanitizer.sanitize(entry.name)
-            return AppFile(name, FileType.RESOURCE, entry.downloadSize, entry.installSize)
+            val resourceType: ResourceType = when {
```

Comment:
The types are re-assigned further down the chain anyways (see `TypeAssigningBucket`) - so we can just omit the `resourceType` here to avoid duplicating code.

Reply:
Hmmm, once I remove the resourceType assigning here the resource report is empty.

Reply:
Just had a quick look. I thought that each file runs through all the buckets, but turns out I remembered it wrong and we only find the first bucket that's applicable and then only runs through that one (see [here](https://github.com/spotify/ruler/blob/main/ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkSanitizer.kt#L52-L56)). So that makes sense, let's keep it like it is now then - sorry for the confusion.

Reply:
No worries! I'm still learning the code not entierly sure how things are working.